docs/content/en/contribute/releasing/_index.md

```diff
@@ -6,64 +6,502 @@ weight: 11
+git checkout main
+git pull ${GIT_REMOTE?} main
+```
+
+### Create the release branch
+
+```shell
+git checkout -b release/${RELEASED_MAJOR_MINOR?}
+```
+
+### Update the changelog
+
+Edit `CHANGELOG/CHANGELOG-${RELEASED_MAJOR_MINOR?}.md` and replace the `## unreleased` section with the new release
+version and date (keep the `## unreleased` section above it).
+
+This should look similar to this:
+
+```diff
+diff --git a/CHANGELOG/CHANGELOG-1.6.md b/CHANGELOG/CHANGELOG-1.6.md
+index 92e9647..a42e086 100644
+--- a/CHANGELOG/CHANGELOG-1.6.md
++++ b/CHANGELOG/CHANGELOG-1.6.md
+@@ -15,6 +15,8 @@ When cutting a new release, update the `unreleased` heading to the tag being gen
+
+ ## unreleased
+
++## v1.6.0 - 2023-03-10
++
+ * [CHANGE] [#907](https://github.com/k8ssandra/k8ssandra-operator/issues/907) Update to cass-operator v1.15.0, remove Vector sidecar, instead use cass-operator's server-system-logger Vector agent and only modify its config
+```
+
+Commit the changes:
+```shell
+git add CHANGELOG/CHANGELOG-${RELEASED_MAJOR_MINOR?}.md
+git commit -m"Update changelog for v${RELEASED_MAJOR_MINOR?}.0"
+```
+
+### Check that the documentation builds
+
+```shell
+cd docs
+hugo mod clean
+npm run build:production
+
+cd ..
+```
+
+If there are any errors, fix them and commit them on the release branch.
+
+### Create the release commit
+
+Update the image tag in the default kustomization:
+```shell
+yq e -i '.images[0].newTag="v"+env(RELEASED_MAJOR_MINOR)+".0"' config/deployments/default/kustomization.yaml
+```
+
+This should look similar to this:
+```diff
+diff --git a/config/deployments/default/kustomization.yaml b/config/deployments/default/kustomization.yaml
+index a06412e..e43b28a 100644
+--- a/config/deployments/default/kustomization.yaml
++++ b/config/deployments/default/kustomization.yaml
+@@ -8,5 +8,4 @@ resources:
+
+ images:
+   - name: k8ssandra/k8ssandra-operator
+-    newTag: latest
+-
++    newTag: v1.6.0
+```
+
+Update the version in the release chart:
+```shell
+yq e -i '.version=env(RELEASED_MAJOR_MINOR)+".0"' charts/k8ssandra-operator/Chart.yaml
+```
+
+This should look similar to this:
+```diff
+diff --git a/charts/k8ssandra-operator/Chart.yaml b/charts/k8ssandra-operator/Chart.yaml
+index ce8f7d1..f32040d 100644
+--- a/charts/k8ssandra-operator/Chart.yaml
++++ b/charts/k8ssandra-operator/Chart.yaml
+@@ -3,7 +3,7 @@ name: k8ssandra-operator
+ description: |
+   Kubernetes operator which handles the provisioning and management of K8ssandra clusters.
+ type: application
+-version: 1.6.0-SNAPSHOT
++version: 1.6.0
+ dependencies:
+   - name: k8ssandra-common
+     version: 0.29.0
+```
+
+Commit and push the changes:
+```shell
+git add config/deployments/default/kustomization.yaml charts/k8ssandra-operator/Chart.yaml
+git commit -m"Release v${RELEASED_MAJOR_MINOR?}.0"
+git push ${GIT_REMOTE?} release/${RELEASED_MAJOR_MINOR?}
+```
+
+Wait for the [GitHub Actions](https://github.com/k8ssandra/k8ssandra-operator/actions) workflows to complete.
```

Comment:
Are e2e tests supposed to pass at this stage? (the tag doesn't exist yet)
If not it should be documented.